core/src/main/java/hudson/model/LargeText.java

```diff
@@ -81,6 +81,7 @@ public boolean exists() {
         this.completed = completed;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Why are you bothering? This whole class is `@Deprecated` so javac should not be emitting any deprecation warnings from it to begin with.

Reply:
I'm pretty sure I only added suppressions because of warnings.

Reply:
Recheck, should be unnecessary.